packages/teams-js/src/public/video.ts

```diff
@@ -134,7 +134,7 @@ export namespace video {
   /**
    * @beta
    * Namespace to get video frames from a media stream.
-   * When the host supports this capability, developer should call {@link mediaStream.registerForVideoFrame} to get the video frames instead of {@link sharedFrame.registerForVideoFrame}, callback of {@link sharedFrame.registerForVideoFrame} will be ignored when the host supports this capability.
+   * When the host supports this capability, developer should call {@link mediaStream.registerForVideoFrame} to get the video frames.
```

Comment:
> I know that your plan (currently) is that mediaStream and sharedFrame would never both be supported at the same time. 

Yes

> In case that changes in the future, it may be worth adding a comment to mediaStream saying that if both mediaStream and sharedFrame are supported the developer should use mediaStream.

I can't predict what change will be in the future, so I prefer not to add such comments at the moment.